scripts/run_vsim.sh

```diff
@@ -31,3 +31,4 @@ done
 call_vsim tb_axi_delayer
 call_vsim tb_axi_atop_filter -GN_TXNS=1000
 call_vsim tb_axi_xbar -t 1ns -coverage -voptargs="+acc +cover=bcesfx"
+call_vsim tb_axi_lite_mailbox -t 1ns -coverage -voptargs="+acc +cover=bcesfx"
```

Comment:
I could not run the testbench because `axi_test::rand_axi_lite_master` is missing. Please rebase on the current master branch.